warehouse/packaging/models.py

```diff
@@ -358,6 +359,19 @@ def urls(self):
 
         return _urls
 
+    @property
+    def github_repo_info_url(self):
+        if self.home_page:
+            parsed = urlparse(self.home_page)
+            if parsed.netloc == 'github.com' and parsed.path.count('/') >= 2:
+                splited_path = parsed.path.split('/')
+                user_name, repo_name = splited_path[1], splited_path[2]
+                tpl = "https://api.github.com/repos/{user_name}/{repo_name}"
+                return tpl.format(
+                    user_name=user_name,
+                    repo_name=repo_name
+                )
```

Comment:
It is possible for users to specify multiple project URLs, any of which could be links to a github repo. Could you update this to look at all the URLs in `self.urls`, not just `self.home_page`?

Reply:
Also I'm not sure what `tpl` means here, perhaps this could just be:

```python
return f"https://api.github.com/repos/{user_name}/{repo_name}"
```